tools/build_all.bat

```diff
@@ -0,0 +1,77 @@
+REM this file, when run from cmake folder, will re-re-re-re-re build the project in all combinations of GBALLOC_HL_TYPE and GBALLOC_LL_TYPE and run all tests for debug and release.
+REM very useful on dev machine
+
+cmake . -DGBALLOC_HL_TYPE:STRING=PASSTHROUGH -DGBALLOC_LL_TYPE:STRING=PASSTHROUGH

```

Comment:
> . -DGBALLOC_HL_TYPE:STRING=PASSTHROUGH -DGBALLOC_LL_TYPE:STR [](start = 5, length = 61)

We should have this in the gate. "If its not in the gate it does not exist".
I think at least at the level of this repo (to prove the switches work) we should have a template yml with parameters and spawn a leg for each config and run it.

Reply:
Great idea, but PHLEASE not this iteration. Here's a task I will do next about it 

[https://msazure.visualstudio.com/One/_workitems/edit/7843114](https://msazure.visualstudio.com/One/_workitems/edit/7843114)

---
In reply to: [462450717](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462450717) [](ancestors = 462450717)